lib/Test/MockObject.pm

```diff
@@ -158,6 +158,16 @@ sub called
     return 0;
 }
 
+sub num_of_calls
+{
+    my ($self, $sub) = @_;
+
+    my $count = grep { $_->[0] eq $sub }
+        reverse @{ _calls( $self ) };
```

Comment:
Wrote this code over 6 months ago so had to re-look at this code. I agree, I don't see the point in doing the work in reversing the list. I've amended the commit and removed this line. Good spot. 👍